core/src/code.rs

```diff
@@ -398,10 +418,20 @@ impl Code {
         }
 
         if let Some(stack_limit) = config.stack_height {
+            let globals = config.export_stack_height.then(|| module.globals_space());
```

Comment:
we must take care that this is never used in prod because of the fact that such change requires re-intrumentation (!)